### PR TITLE
Preparing for 1.0.0

### PR DESCRIPTION
**Pull request by gmendonca**
Creating version 1.0.0. The intent here is to add version support to tiopatinhas so further improvements could be organized in different version tags.

### CODE REVIEW

**Comment by gmendonca** [comment]
@barata @raittes @mwconceicao

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+## 1.0.0 (October 11, 2016)
+    - Added Monitoring enabled option
+    - Added submodule support
+    - Added VPC support
+    - Added instance IAM rol /profile support
```
Comment (barata) [inline CHANGELOG.md]: rol?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
...
+    - Removed CPU checks from tiopatinhas and added weight factor (now you decide the ratio of machines that tiopatinhas will handle and it let's ASG handles the number of machines)
```
Comment (barata) [inline CHANGELOG.md]: "it let's" sounds strange.

File: setup.py [hunk 3]
```diff
@@ -1,10 +1,19 @@
...
+    description='An Amazon Autoscaling companion that uses the Spot Market',
+    license='Open Source',
+    install_requires=requires,
+    packages=['tp'],
+    package_data={'tp': ['tp.conf']},
+    long_description=open('README.md').read(),
+    zip_safe=False,
+    url='https://www.github.com/chaordic/tiopatinhas/',
+    scripts=['tp/tp.py']
+)
```
Comment (mwconceicao) [inline setup.py]: Missing `\n` at the end

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,11 @@
+## 1.0.0 (October 11, 2016)
+    - Added Monitoring enabled option
+    - Added submodule support
+    - Added VPC support
+    - Added instance IAM role /profile support
```
Comment (mwconceicao) [inline CHANGELOG.md]: How about `role/profile` or `role / profile` ?

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,11 @@
...
+    - Removed CPU checks from tiopatinhas and added weight factor (now you decide the ratio of machines that tiopatinhas will handle and the ASG handles everything else)
+    - Fixes on Market crashed handlers and LB bugs
+
```
Comment (mwconceicao) [inline CHANGELOG.md]: The use of `marked-for-termination` status to set an instance as not live might be worth of mentioning here?

**Pull request merged**